solutions/swb-reference/integration-tests/support/clientSession.ts

```diff
@@ -45,6 +46,14 @@ export default class ClientSession {
       headers
     });
 
+    this._axiosInstance.interceptors.request.use((config) => {
+      config.paramsSerializer = (params) => {
+        return Qs.stringify(params);
+      };
+
+      return config;
+    });
```

Comment:
this is a known issue with nested params and axios, commonly fixed with qs library